nsv13/code/modules/overmap/physics.dm

```diff
@@ -494,7 +494,7 @@ The while loop runs at a programatic level and is thus separated from any thrott
 		velocity.x -= bump_impulse
 	return ..()
 
-/obj/structure/overmap/Bump(atom/movable/A, datum/collision_response/c_response)
+/obj/structure/overmap/Bump(atom/movable/A, datum/collision_response/c_response, atom/movable/B)
```

Comment:
src and B should be equal, so you can get rid of the atom/movable/B and replace all B. things with either src. or just nothing.